dhis-2/dhis-api/src/main/java/org/hisp/dhis/feedback/ErrorCode.java

```diff
@@ -157,7 +157,7 @@ public enum ErrorCode {
   E2210("Data end date not allowed"),
   E2211("Algorithm min-max values not allowed"),
   E2212("Specifying both a start date/end date and a relative period is not allowed"),
-
+  E2213("Order by value is not compatible with algorithm"),
```

Comment:
could we add `{0}` to return which algorithm ?

Reply:
```suggestion
  E2213("Value of param orderBy is not compatible with algorithm `{0}`"),
```

Reply:
done